sensor/kubernetes/enforcer/cronjob/suspend.go

```diff
@@ -0,0 +1,62 @@
+package cronjob
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/branding"
+	kubernetesPkg "github.com/stackrox/rox/pkg/kubernetes"
+	"github.com/stackrox/rox/pkg/retry"
+	"github.com/stackrox/rox/sensor/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	batchV1      = "batch/v1"
+	batchBeta1v1 = "batch/beta1v1"
+)
+
+// Suspend suspends the cron job
+func Suspend(ctx context.Context, client kubernetes.Interface, deploymentInfo *central.DeploymentEnforcement) (err error) {
+	depName := deploymentInfo.GetDeploymentName()
+	depType := deploymentInfo.GetDeploymentType()
+	forcePatch := true
+
+	if ok, apiErr := utils.HasAPI(client, batchV1, kubernetesPkg.CronJob); ok && apiErr == nil {
+		patch := fmt.Sprintf("{\"metadata\": {\"name\": %s}, \"kind\": %s, \"apiVersion\": %s, \"spec\": {\"suspend\": true}}", depName, depType, batchV1)
+
+		_, err = client.BatchV1().CronJobs(deploymentInfo.GetNamespace()).Patch(ctx, depName, types.ApplyPatchType,
+			[]byte(patch),
+			metav1.PatchOptions{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       depType,
+					APIVersion: batchV1,
+				},
+				FieldManager: branding.GetProductName(),
```

Comment:
I think it should be sufficient to say `"StackRox"` here instead of `branding.GetProductName()`.
Judging that it was ok to use `StackRox` in https://github.com/stackrox/stackrox/blob/9bdc76a0e18b85b4439cfce1c48844015520b0d5/pkg/detection/deploytime/utils.go#L14-L14

Reply:
Why? I'd say we change that in the "BLockedBy..." to reflect product name. We don't want StackRox to leak through the RH branding right?

Reply:
> we change that in the "BLockedBy..."

This was just one nearby instance of many hundreds of others that mention `stackrox`. I don't think we can change that thing exposed in settings, environment variables, k8s objects labels and annotations that quickly.
Hence, I don't see a reason to make this spot any special and use branded name here.

My main concern about usage of branded name as a field manager is the restrictions that are put on the value
https://github.com/kubernetes/apimachinery/blob/6f5782f4e5f2798853afeb749157008669784e83/pkg/apis/meta/v1/types.go#L536-L541

In case the branded name gets too long, in case it gets some special characters (©, ™) or funny emoji, the code starts breaking.

I don't see a point to commit to such a time bomb especially without sufficient test coverage.

Reply:
Changed, though I believe we are regressing. Funny characters and emojis should be within our control in branding.

Reply:
I don't believe we, actually the person adjusting branding in the future, will remember about all the nuances of what's allowed or what's in the entire stack should be verified when changing the branding. Especially in the absence of end-to-end tests that would automatically catch regressions.